src/audio/audio.cpp

```diff
@@ -36,6 +37,37 @@
 #endif
 
 /**
+@internal
+
+@class Audio::Private
+
+@brief Encapsulates private audio framework from public qTox Audio API.
+*/
+class Audio::Private
+{
+public:
+    Private()
+        : minInputGain{-30.0}
+        , maxInputGain{30.0}
+        , gain{0.0}
+        , gainDecibel{0.0}
+    {
+    }
+
+    void setInputGain(qreal dB)
+    {
+        gainDecibel = qBound(minInputGain, dB, maxInputGain);
+        gain = qPow(10.0, (dB * 0.05));
```

Comment:
The parameter `gain` is being calculated from the unbounded input argument `dB`, resulting in incorrect behaviour when out of bounds values are used (i.e. whilst `gainDecibel` will correctly clip to +-30 dB, `gain` will be arbitarily set based on the input dB value, which could potentially come from wrong/outdated settings).

Not an issue but, is there a particular reason to use `dB * 0.05` instead of `dB / 20.0`? I feel the latter is more clear.